src/components/Button.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../css/modules/Button.module.css';
+
+const generateClassName = (buttonIsWide, stylesObject) => {
+  const { button, wide } = stylesObject;
+  if (buttonIsWide) {
+    return `${button} ${wide}`;
+  }
+  return button;
+};
+
+const Button = ({ name, isWide, color, onClick }) => (
+  <button
+    style={{ backgroundColor: color }}
+    className={generateClassName(isWide, styles)}
+    type="button"
+    id={`button-${name}`}
+    onClick={onClick}
+  >
+    {name}
+  </button>
+);
+
```

Comment:
Meaning that you will have to change this 'onClick' prop names here too

Reply:
See this: https://github.com/bolah2009/react-calculator/pull/5#discussion_r347764590